src/MessageQueue/MessageQueue.php

```diff
@@ -109,6 +107,12 @@ private function doFlush(bool $async)
             foreach ($messagesToPush as $key => $message) {
                 $messageParameters = [
                     'Id'           => $key, // Identifier of the message in the batch
+                    'MessageAttributes' => [
+                        'name' => [
+                            'DataType'    => 'String',
+                            'StringValue' => $message['name'],
```

Comment:
I love how convoluted AWS API can be sometimes. Why a "DataType" => "String", if the attribute is already called "StringValue" :p.